builtin/logical/ssh/path_issue_sign.go

```diff
@@ -101,15 +98,19 @@ func (b *backend) pathSignIssueCertificateHelper(ctx context.Context, req *logic
 		return logical.ErrorResponse(err.Error()), nil
 	}
 
-	privateKeyEntry, err := caKey(ctx, req.Storage, caPrivateKey)
+	// TODO (gabrielopesantos): Review
```

Comment:
I think this makes sense. Did you have thoughts about what you wanted to review?

Reply:
I must have addressed whatever I wanted to review and forgot to review the comment.